### ISSUE TITLE
Tests failing since b58f4da

### ISSUE THREAD

**Issue by rwjblue** [knownasilya/ember-query-params#13]
2 test failures were introduced by b58f4da. From [Travis build logs](https://travis-ci.org/knownasilya/ember-query-params/jobs/122260349#L305):

```
not ok 16 PhantomJS 1.9 - controller:test: it works
    ---
        actual: >
            null
        stack: >
            TypeError: 'undefined' is not an object (evaluating 'relay.setParam')
        message: >
            Died on test #1     at test (http://localhost:7357/assets/test-support.js:3182)
                at testWrapper (http://localhost:7357/assets/test-support.js:6551)
                at test (http://localhost:7357/assets/test-support.js:6564)
                at http://localhost:7357/assets/tests.js:187
                at http://localhost:7357/assets/vendor.js:92
                at http://localhost:7357/assets/vendor.js:142
                at findModule (http://localhost:7357/assets/vendor.js:190)
                at http://localhost:7357/assets/vendor.js:177
                at http://localhost:7357/assets/test-loader.js:67
                at http://localhost:7357/assets/test-loader.js:58
                at http://localhost:7357/assets/test-loader.js:89
                at http://localhost:7357/assets/test-support.js:6383: 'undefined' is not an object (evaluating 'relay.setParam')
        Log: |
    ...
not ok 17 PhantomJS 1.9 - controller:test: auto unsubscribe
    ---
        actual: >
            null
        stack: >
            TypeError: 'undefined' is not an object (evaluating 'relay.subscribe')
        message: >
            Died on test #1     at test (http://localhost:7357/assets/test-support.js:3182)
                at testWrapper (http://localhost:7357/assets/test-support.js:6551)
                at test (http://localhost:7357/assets/test-support.js:6564)
                at http://localhost:7357/assets/tests.js:223
                at http://localhost:7357/assets/vendor.js:92
                at http://localhost:7357/assets/vendor.js:142
                at findModule (http://localhost:7357/assets/vendor.js:190)
                at http://localhost:7357/assets/vendor.js:177
                at http://localhost:7357/assets/test-loader.js:67
                at http://localhost:7357/assets/test-loader.js:58
                at http://localhost:7357/assets/test-loader.js:89
                at http://localhost:7357/assets/test-support.js:6383: 'undefined' is not an object (evaluating 'relay.subscribe')
        Log: |
    ...
```

**Comment by knownasilya** [comment]
Fixed in 2.0

**Comment by rwjblue** [comment]
Awesome, thank you!
